infra/Pos/DHT/Real/Real.hs

```diff
@@ -147,9 +147,13 @@ kademliaGetKnownPeers
     -> m [DHTNode]
 kademliaGetKnownPeers inst = do
     let myId = kdiKey inst
-    let explicitInitial = kdiExplicitInitial inst
-    let initPeers = bool [] (kdiInitialPeers inst) explicitInitial
-    buckets <- liftIO (K.viewBuckets $ kdiHandle inst)
+    let kInst = kdiHandle inst
+    let initNetAddrs = map toKPeer (kdiInitialPeers inst)
+    initPeers <- map toDHTNode <$>
+                     bool (pure [])
+                     (catMaybes <$> liftIO (K.peersToNodeIds kInst initNetAddrs))
```

Comment:
`peersToNodeIds` will return `[]` if node just initialized and there were no dump. We should use different here. If this is the case that we need return peers as `[DHTNode]`, then my proposal is to rewrite this method to use `[NetworkAddress]` and instead of `(K.peersToNodeIds kInst initNetAddrs)` have `initNetAddrs`